epicor/README.md

```diff
@@ -0,0 +1,44 @@
+---------------------------
+Author Name:IanGSalmon
+**Epicor10 Element**  
+This element allows the creation of an element instance for the Epicor10, on-prem API. In order to implement this element, you will need a method for exposing the local network with access to the Epicor10 API, such as Ground2Cloud.  
+#  
```

Comment:
@IanGSalmon Can you provide a link to the vendor's docs in here? Also, maybe link to our Ground to Cloud docs?

Reply:
Added